src/app/pages/crag/crag-routes/crag-routes.component.ts

```diff
@@ -140,9 +142,25 @@ export class CragRoutesComponent implements OnInit, OnDestroy {
     private localStorageService: LocalStorageService,
     private changeDetection: ChangeDetectorRef,
     private hostElement: ElementRef,
-    private searchService: SearchService
+    private searchService: SearchService,
+    private dialog: MatDialog
   ) {}
 
+  openDialog(routeId, routeName) {
```

Comment:
I suggest adding preventDefault call to prevent opening/closing accordion when opening the modal (similar to checkbox event from the beginning of the row).
Something like this works for me:
```
  openDialog(event, routeId, routeName) {
    event.stopPropagation();
```

Reply:
Done.